composer.json

```diff
@@ -107,7 +107,7 @@
         "nyholm/psr7": "^1.2",
         "orchestra/testbench-core": "^8.23.4",
         "pda/pheanstalk": "^4.0",
-        "phpstan/phpstan": "^1.4.7",
+        "phpstan/phpstan": "~1.11.11",
```

Comment:
Since we no longer releasing feature or bugfixes, we should lock phpstan/phpstan to the version last known to work based on `phpstan.src.neon.dist` and `phpstan.types.neon.dist`.